ci/build/build-docker-images.py

```diff
@@ -424,6 +424,7 @@ def prep_ray_ml():
 
     requirements_files = [
         "python/requirements.txt",
+        "python/requirements_compiled.txt",
```

Comment:
could you point out to me where does the `requirements_compiled.txt` come from? I was expecting this file to be checked in into the repo, but did not find it.

Reply:
It's compiled in `ci.sh`